src/utilities/browser.ts

```diff
@@ -16,3 +16,8 @@ export function isSafari() {
         (navigator.userAgent.indexOf("Chrome") < 0) &&
         (navigator.userAgent.indexOf("Chromium") < 0);
 }
+
+export function isMac() {
+  return typeof window !== "undefined" &&
+         /Mac|iPod|iPhone|iPad/.test(window.navigator.platform);
```

Comment:
I was thinking about that, but I thought, as a utility, it could be useful if it were more general. I even thought of making the name isApple() and a 2nd predicate called isAppleMobile(). I figured leaving the RE with the 4 options would provide a hint as to how this could be modified, in the future. One more thing, I didn't research the keyboards on the iPadPro. Any idea if they support CMD keys?